src/utils/Banner.js

```diff
@@ -0,0 +1,22 @@
+///>>> Creating a Correct Answer Banner
+import { USER_INTERFACE_ID } from '../constants.js';
+const userInterface = document.getElementById(USER_INTERFACE_ID);
+
+export function correctBanner() {
+  const correctAnswerBanner = document.createElement('p');
+  correctAnswerBanner.textContent = 'Correct Answer';
+  correctAnswerBanner.style.display = 'block';
```

Comment:
Can you add this style in css?

Reply:
I noticed that this line is not needed anymore, so I am deleting it